README-pt_br.md

```diff
@@ -0,0 +1,110 @@
+# react-howto
+
+Se você é novo no React (ou no frontend em geral) você pode achar o ecossistema confuso. Existem algumas razões para isso.
+
+* React tem sido historicamente orientado para iniciantes e especialistas
+* Facebook apenas abre o código fonte daquilo que ele realmente utiliza, então ele não se concentra em ferramentas para projetos menores que o Facebook.
+* Há muito marketing ruim que aparecem como guias para o React.
+
+Ao longo deste documento, eu irei assumir que você já construiu alguma página Web com HTML, CSS e JavaScript.
+
+## Por que você deve me escutar?
+
+Há uma tonelada de conselhos conflitantes sobre o React por aí; por que me ouvir?
+
+Eu fui um dos membros originais da equipe do Facebook que construiu e abriu o código fonte do React. Eu não estou mais no Facebook e sim em uma pequena startup, então eu também tenho uma perspectiva de fora do Facebook.
+
+## Como lidar com o ecossistema do React
+
+Todo software é construído sobre uma pilha de tecnologias, e você precisa entender o suficiente dessa pilha para construir sua aplicação. A razão pela qual o ecossistema de ferramentas do React parece te esmagar é porque ele é sempre explicado na ordem errada.
+
+Você deve aprender, nessa ordem, **sem pular para o próximo ou aprender simultaneamente**:
+
+* [React em si](#learning-react-itself)
+* [`npm`](#learning-npm)
+* [JavaScript “bundlers”](#learning-javascript-bundlers)
+* [ES6](#learning-es6)
+* [Rotas](#learning-routing)
+* [Flux](#learning-flux)
+
+**Você não precisa aprender todos para ser produtivo com o React.** Somente pule para o próximo passo caso você esteja com algum problema que necessite ser resolvido.
```

Comment:
Embora *todos*, como foi usado, dê uma ideia de conjunto, a ideia do autor ("all of these") parece estar mais para amplificar o peso de aprender o conjunto todo. Assim, "tudo" (como meu professor dizia: um indefinido definido) soa melhor e fica mais próximo do original